app/src/openxr/cpp/OpenXRInputSource.cpp

```diff
@@ -605,7 +608,7 @@ void OpenXRInputSource::Update(const XrFrameState& frameState, XrSpace localSpac
     UpdateHaptics(delegate);
 }
 
-XrResult OpenXRInputSource::UpdateInteractionProfile()
+XrResult OpenXRInputSource::UpdateInteractionProfile(ControllerDelegate& controller)
```

Comment:
Nit, can we call this delegate? I think it better explains that the object is and is consistent with the rest of the file.

Reply:
Ok. FWIW, I think `controller` is less ambiguous than `delegate` in this case, but true that rest of code uses the latter. Fixed.